front/src/components/AuthorTheCheat.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { StyleSheet, Text, TouchableOpacity } from "react-native";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+import { useRecoilState } from "recoil/dist";
+import { authorTheCheatState } from "../states/memberState";
+
+export default function AuthorTheCheat() {
+  const [theCheatState, setTheCheatState] = useRecoilState(authorTheCheatState);
+
+  const iconName = theCheatState ? "account-check" : "account-check-outline";
+  const iconColor = theCheatState ? "green" : "grey";
+  const pressTheCheat = () => {
+    if (theCheatState) {
+      setTheCheatState(false);
+      return;
+    }
+    setTheCheatState(true);
+  };
```

Comment:
삼항연산자를 이용하면 더욱 깔끔해질 것 같아요! 
```suggestion
  const pressTheCheat = () => {
    theCheatState ? setTheCheatState(false) : setTheCheatState(true);
  };
```